src/util/vfs/vfs-zip.c

```diff
@@ -451,7 +452,8 @@ static enum VFSType _vdezType(struct VDirEntry* vde) {
 bool _vfzClose(struct VFile* vf) {
 	struct VFileZip* vfz = (struct VFileZip*) vf;
 	unzCloseCurrentFile(vfz->z);
-	free(vfz->buffer);
+	if(vfz->buffer)
```

Comment:
Does not match style. Single line ifs must have braces, if statements need a space between the keyword and parens..